system.c

```diff
@@ -176,6 +176,14 @@ uint8_t system_execute_line(char *line)
           else { report_ngc_parameters(); }
           break;
 
+        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+          char_counter++;
+          if (!read_float(line, &char_counter, &parameter)){
```

Comment:
`read_float` doesn't cast from a `char` to a `float`. It computes the `float` representation based on the ascii values in the `line` buffer.